### PR TITLE
Add Cameras

### PR DESCRIPTION
**Pull request by szhu**

### CODE REVIEW

**Comment by szhu** [comment]
I couldn't find a better homepage, because I don't know where to look for it.

File: Casks/cameras.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+cask :v1 => 'cameras' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.flexibits.com/cameras/downloads/Cameras.dmg'
+  name 'Cameras'
+  # Flexibits no longer has a homepage for Cameras
+  homepage 'http://www.macupdate.com/app/mac/31606/cameras'
```
Comment (vitorgalvao) [inline Casks/cameras.rb]: Couldn’t find something on the wayback machine?

Comment (szhu) [inline Casks/cameras.rb]: I didn't know where to look. I tried http://www.flexibits.com/cameras and it wasn't there.

Comment (vitorgalvao) [inline Casks/cameras.rb]: [There is one](https://web.archive.org/web/20120807002827/http://www.flexibits.com/cameras), it just appears to be down. You can switch it up.

File: Casks/cameras.rb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+cask :v1 => 'cameras' do
+  version :latest
+  sha256 :no_check
+
+  url 'http://www.flexibits.com/cameras/downloads/Cameras.dmg'
+  name 'Cameras'
+  # Flexibits no longer has a homepage for Cameras
```
Comment (vitorgalvao) [inline Casks/cameras.rb]: The comment should go.

File: Casks/cameras.rb [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+  homepage 'http://www.macupdate.com/app/mac/31606/cameras'
+  license :unknown    # todo: change license and remove this comment; ':unknown' is a machine-generated placeholder
```
Comment (vitorgalvao) [inline Casks/cameras.rb]: `:gratis`.

File: Casks/cameras.rb (hunk 1 above)
Comment (szhu) [inline Casks/cameras.rb]: That link intermittently works. I tried it and it looked like a page of ads.

**Comment by szhu** [comment]
Fixed everything except the homepage.

**Comment by vitorgalvao** [comment]
Looks like this one will have to wait on https://github.com/caskroom/homebrew-cask/issues/5248.

**Comment by szhu** [comment]
Hm, I do feel like the macupdate page is useful as it provides a description and links to the download link given. Changing the homepage to `:unknown` would remove this useful information. Is this intentional?

**Comment by vitorgalvao** [comment]
Well, since it’s in this repo anyway, why not.

**Comment by szhu** [comment]
Thanks!

**Pull request merged**